backend/src/server.ts

```diff
@@ -72,8 +72,8 @@ class Server {
         Logger.warn('HTTPS SERVER START FAILED', { error, details: error.toString(), directory: WEB_DIR })
         app.quit()
       })
-      .listen(SSL_PORT, () => {
-        Logger.info('HTTPS SERVER STARTED', { port: WEB_PORT, directory: WEB_DIR })
+      .listen(SSL_PORT, HOST, () => {
```

Comment:
Actual bugs I noticed. The HOST was important because it controls a setting to disable local discovery.